scss/_patterns_buttons.scss

```diff
@@ -117,17 +174,63 @@ $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
     @include vf-focus($color-focus-negative);
   }
 
+  %p-button--negative-dark {
+    $color-negative-dark: adjust-color($color-negative, $lightness: -10%);
```

Comment:
@lyubomir-popov This is just provisional darker version of negative color, I'll let you adjust it properly.

Reply:
add !default; maybe?